websites/D/DCGO/presence.ts

```diff
@@ -0,0 +1,54 @@
+const presence = new Presence({
+	clientId: "1075115687888044072",
+});
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/NPVqgsx.png",
+	};
+
+	const { pathname } = document.location;
```

Comment:
```suggestion
	},
	 { pathname } = document.location,
	pathnameLowercase = pathname.toLowerCase();
```

Reply:
i updated

Reply:
Don't forget to add this otherwise your changes won't work.

Reply:
I don't know what you're talking about.

Reply:
Oh, okay, I get it.